code/datums/components/onwear_mood.dm

```diff
@@ -0,0 +1,50 @@
+// Add to clothing to give the wearer a mood buff and a unique examine str
+
+/datum/component/onwear_mood
+	/// the event the wearer experiences
+	var/datum/mood_event/saved_event
+	/// examine string added to examine
+	var/examine_string
+
+/datum/component/onwear_mood/Initialize(clear_after, datum/mood_event/saved_event, examine_string)
```

Comment:
`clear_after` should be the last argument, as this component doesn't look like it NEEDS it as much as it does something like saved_event.